runtime/lib/capi/RuntimeCAPI.cpp

```diff
@@ -1036,14 +1035,34 @@ int64_t __catalyst__rt__array_get_size_1d(QirArray *ptr)
     return qubit_vector_ptr->size();
 }
 
+static bool isDeviceAutomaticQubitManageMode()
```

Comment:
Function looks a bit superfluous to me (length is about the same especially after cutting the name on the class, clarity is the same, ...)